js_modules/dagster-ui/packages/ui-core/src/assets/PartitionHealthSummary.tsx

```diff
@@ -19,6 +21,37 @@ export const PartitionHealthSummary = memo((props: Props) => {
   const assetData = data.find(
     (d) => JSON.stringify(d.assetKey.path) === JSON.stringify(assetKey.path),
   );
+  const history = useHistory();
+
+  const handlePartitionInteraction = useCallback((selectedPartitions: string[], dimension?: PartitionHealthDimension) => {
+
+    if (selectedPartitions.length > 0) {
+      const firstPartition = selectedPartitions[0]!;
+      const lastPartition = selectedPartitions[selectedPartitions.length - 1]!;
```

Comment:
We just added linting to prevent using non-null assertions. Please handle the null/undefined cases. :)